frontend/src/components/ExperimentCollection/ExperimentCollection.tsx

```diff
@@ -20,7 +20,8 @@ interface ExperimentCollectionProps extends RouteComponentProps<RouteParams> {
 
 const ExperimentCollection = ({ match }: ExperimentCollectionProps) => {
     const [experimentCollection, loadingExperimentCollection] = useExperimentCollection(match.params.slug) as [IExperimentCollection, boolean];
-    const experimentToRedirectTo = experimentCollection?.redirect_to;
+    const nextExperiment = experimentCollection?.next_experiment;
+    const displayDashboard = experimentCollection?.dashboard.length;
```

Comment:
I think the PR description now says this is always an array. Does that mean this always resolves in `true`?

Reply:
It's either 0 (=false), or a positive integer (=true).

Reply:
Ahhhh yes, one of the infamous javascript features 😊